packages/polyfills/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+- Replace deprecated `babel-polyfills` in favor of `core-js` and `regenerator-runtime`. [#1660](https://github.com/Shopify/quilt/pull/1660)
```

Comment:
Would this be a breaking change or minor?

Reply:
I think it would actually be a major since it requires consumers to run babel with the corejs v3 flag. But @alexandcote shared worries with such an upgrade as he tried to do a the same upgrade in sewing kit https://github.com/Shopify/sewing-kit/pull/1899 So I'm not exactly sure what the foundation team decided to do with this core-js upgrade

Reply:
I'm planning to do the full core-js bump during the next hack day so after BFCM most of the project will be able to use core-js 3 and new ES features.